github/review/README.md

```diff
@@ -0,0 +1,65 @@
+# Mass GitHub Review from Command Line
+
+Sometimes you need to review a lot of pull requests with the same or similar
+small change. Using the GitHub web UI is not convenient for approving many pull
+requests, for that purpose we have this `review.rb` helper script which can be
+used from command line.
+
+## Features
+
+- Displays details of each pull request (title, diff)
+- Displays also the check results (Travis, etc...) to avoid merging a pull
+  request which does not pass CI
+- Can approve each pull request with the default "LGTM" message
+- Optionally can merge the pull requests and delete the source branch
+
+If a pull request is not approved then it is skipped. You need to comment
+at the GitHub pull request page manually in that case.
+
+## Authentication
+
+You need to [create a GitHub access token](https://github.com/settings/tokens)
+to enable GitHub access from scripts.
+
+### Using .netrc File
+
+You can store the generated token into the `~/.netrc` file, see the
+[this documentation](https://github.com/octokit/octokit.rb#using-a-netrc-file)
+for more details.
+
+Just make sure the `.netrc` file is not readable for the other users
+(`chmod 0600 ~/.netrc`).
+
+### Passing via Environment
+
+Alternatively the access token can be passed via environment variable `GH_TOKEN`:
+
+```shell
+␣GH_TOKEN=... ./review ...
+```
+
+*Note the extra space at the beginning of the line, that is required to
+avoid storing the command with the token into the shell history file
+(like `~/.bash_history`).*
+
+## Usage
+
+Just pass the list of the pull request URLs as the script arguments, for
+example:
+
+```shell
+./review.rb --merge https://github.com/yast/yast-users/pull/218 \
+  https://github.com/yast/yast-firstboot/pull/76 \
+  https://github.com/yast/yast-add-on/pull/84 \
+  https://github.com/yast/yast-sysconfig/pull/29 \
+  https://github.com/yast/yast-sound/pull/35
```

Comment:
I think it would be nice to be able to read it from file that list. It is much more usable for me.